extensions/amp-analytics/0.1/variables.js

```diff
@@ -27,6 +28,13 @@ const TAG = 'amp-analytics/variables';
 /** @const {RegExp} */
 const VARIABLE_ARGS_REGEXP = /^(?:([^\s]*)(\([^)]*\))|[^]+)$/;
 
+const EXTERNAL_CONSENT_POLICY_STATE_STRING = {
+  1: 'accepted',
```

Comment:
So I looked at the `CONSENT_POLICY_STATE`, and realized that maybe I want to rename these values to
```
{
  1: 'sufficient',
  2: 'insufficient',
  3: 'unknown_not_required',
  4: 'unknown'
}
```
Two reasons:
1. To still match the internal `CONSENT_POLICY_STATE`, even though sufficient is directly related to accepted now. 
2. In the case of fallback or auto reject. It's safer to say consent is insufficient compared to user has rejected to give consent. 

@torch2424 @lannka let me know if you have any objection.

Reply:
the new names look good.

shall we change `unknown_not_required` to `not_required`?

Reply:
SG. The idea is that if user has given consent before, user decision is respected even in the case of not_required. 
But I also like the shorter string. I'll use `not_required` and document that it's `not_required` only when the consent state is unknown. Have #21101 to rename